src/model_locator.py

```diff
@@ -0,0 +1,25 @@
+import os
+import dotenv
+
+
+def __load_model_env(model: str) -> str:
+    dotenv.load_dotenv()
+
+    path_from_env_file = os.getenv(model)
+    if path_from_env_file is not None:
+        return path_from_env_file
+
+    path_inside_container = os.getenv(model + '_DEV')
+
+    if path_inside_container is None:
+        raise NameError(f'Unable to locate model from env {model}')
+
+    return str(path_inside_container)
+
+
+def labeling() -> str:
+    return __load_model_env('MODEL_LABELING')
```

Comment:
It seems like every call of this method looks up the paths anew. Since the environment should not change during runtime, caching the model paths could yield (probably negligible) performance gains and make CiRA _blazingly_ fast.

Reply:
I refactored it to be _blazingly fast_ ⚡️

The functions will be called only once:
```python
dotenv.load_dotenv()
LABELING = __load_model_env('MODEL_LABELING')
CLASSIFICATION = __load_model_env('MODEL_CLASSIFICATION')
```